### ISSUE TITLE
Newer VSF crashes on Ipad Pro

### ISSUE THREAD

**Issue by Eelpan** [DivanteLtd/vue-storefront#4794]
When visiting a newer version of a vsf store on the actual device (ipad pro) not emulated some javascript crash happens and the page does not load correctly.

I have tested in several versions of VSF but cant conclude from where it the bug starts, but testing on demo.vuestorefront.io also allows for the issue to appear.

**Comment by pkarw** [comment]
@Fifciu @gibkigonzo can you give it a try on Safari?

**Comment by Fifciu** [comment]
Just checked it on iPad Pro. demo.vuestorefront.io works well. Try to purge the browser's cache and use a private cart.

A few weeks ago we had a similar situation on VSF Slack. One developer had a problem opening VSF on MacOS. I checked it both on Ubuntu and macOS. In the browser's console, I've found errors about not existing .js files. It looked like server would deploying a new version right now or something went wrong during the previous deploy.

I believe the author of the issue has cached bugged deploy and purging cache + incognito is a solution.
